lib/order.rb

```diff
@@ -0,0 +1,55 @@
+require_relative 'menu'
+
+class Order
+  attr_reader :menu, :cost, :items
+  DELIVERY_TIME = 1800
+
+  def initialize(menu = Menu.new)
+    @menu = menu.list
+    @items = {}
+    cost = 0
+  end
+
+  def update(items)
+    manage_order(items)
+    calculate_cost
+  end
+
+  def confirm
```

Comment:
If you get time/ had time, I'd recommend at looking into using the Twilio API. It is a cool feature to add. 
This could easily be something you implement later on if you fancied. But for the moment I can clearly see how the code `confirms` the order and can see where you'd implement it!